src/partials/home/Services.svelte

```
@@ -25,33 +24,40 @@
+            ? 'border-2 border-brand-600 shadow-xl dark:border-brand-700'
+            : 'border border-gray-200 shadow-md dark:border-neutral-800'}">
+          {#if service.isPopular}
+            <p class="mb-3">
+              <span class="inline-flex items-center gap-1.5 rounded-lg bg-brand-100 px-3 py-1.5 text-xs font-semibold uppercase text-brand-800 dark:bg-brand-600 dark:text-white">Most popular</span>
+            </p>
+          {/if}
 
-        <h4 class="-mt-6 h-24 stroke-black text-lg font-medium text-gray-600 dark:text-neutral-200">{@html service.icon}</h4>
-        <span class="text-xl font-bold text-gray-600 dark:text-neutral-200">{service.title}</span>
-        <p class="mt-7 line-clamp-4 text-xs text-gray-500 hover:line-clamp-none dark:text-neutral-500">{service.desc}</p>
+          <h4 class="-mt-6 h-24 stroke-black text-lg font-medium text-gray-600 dark:text-neutral-200">{@html service.icon}</h4>
```

ESLint

[error] `{@html}` can lead to XSS attack.

(svelte/no-at-html-tags)